frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachineTemplate.ts

```diff
@@ -1,18 +1,34 @@
 /* eslint-disable no-await-in-loop */
-import {
-  filterForName,
-  resourceRowsPresent,
-} from '@console/internal-integration-tests/views/crud.view';
-import { VMTemplateConfig, VirtualMachineTemplateModel } from '../utils/types';
+import { VMTemplateConfig, VirtualMachineTemplateModel, VMConfig } from '../utils/types';
 import { ProvisionConfigName } from '../utils/constants/wizard';
 import { Wizard } from './wizard';
 import { KubevirtUIResource } from './kubevirtUIResource';
+import { VirtualMachine } from './virtualMachine';
+import {
+  VMT_ACTION,
+  VM_STATUS,
+  VM_BOOTUP_TIMEOUT_SECS,
+  VM_IMPORT_TIMEOUT_SECS,
+} from '../utils/consts';
+import { detailViewAction, listViewAction } from '../../views/vm.actions.view';
+
+const confirmedActions: VMT_ACTION[] = [VMT_ACTION.Delete];
```

Comment:
https://github.com/openshift/console/pull/5303 should be merged soon, so there will not be a need for this

Reply:
#5303 merged :tada: :balloon: